dist/css/style.css

```diff
@@ -0,0 +1,219 @@
+html, body {
+  margin: 0;
+  padding: 0;
+  font-family: "Calibri", "Arial", "Helvetica Neue", sans-serif;
+  overflow: hidden;
+}
+
+.page {
+  display: grid;
+  column-gap: 8px;
+  padding: 8px;
+  box-sizing: border-box;
+  height: 100vh;
+  width: 100vw;
+  grid-template-columns: auto 20em;
+  grid-template-rows: 3em auto 3em;
+  grid-template-areas:
+    "header header"
+    "canvas sidebar"
+    "footer footer";
+}
+
+header {
+  grid-area: header;
+  display: flex;
+}
+
+@font-face {
+  font-family: 'Copperplate';
+  src: url('/fonts/Copperplate V1.woff2')  format('woff');
```

Comment:
Can you make this a relative URL?  Currently the site isn't dependent on where the files are put, e.g. they could be placed in `/my/funky/dir/` and everything would still work, but using an absolute URL here would require the site to always be at the root directory of the website it's hosted at.

Also, do you mind renaming `Copperplate V1.woff2` to contain no spaces, e.g. `Copperplate_V1.woff2`?